functions.php

```diff
@@ -50,6 +50,34 @@ function c4aa_add_body_class_from_acf_options( $classes ) {
 }
 add_filter( 'body_class', 'c4aa_add_body_class_from_acf_options' );
 
+/**
+ * Add Dashboard Widget
+ * 
+ * A place for specific reference info & instructions about the theme 
+ * for users of the site.
+ * 
+ */ 
+
+
+add_action('wp_dashboard_setup', 'c4aa_custom_dashboard_widgets');
+  
+function c4aa_custom_dashboard_widgets() {
+global $wp_meta_boxes;
```

Comment:
@slambert The contents of the functions should be indented one tab within the curly braces for readability - just an FYI, will approve and merge.